Plan/plugin/build.gradle

```diff
@@ -61,54 +60,59 @@ shadowJar {
     exclude "jakarta/xml/**/*"
     exclude "javassist/**/*"
 
-    relocate('org.slf4j', 'plan.org.slf4j')
+    relocate("org.slf4j", "plan.org.slf4j")
 
     // Unrelocate a package to use platform version of some libraries that were relocated in Plan
     exclude "com/djrapitops/plan/unrelocate/**/*"
-    relocate('com.djrapitops.plan.unrelocate.', '')
+    relocate("com.djrapitops.plan.unrelocate.", "")
 
-    relocate('org.apache', 'plan.org.apache') {
-        exclude 'org/apache/logging/**'
-        exclude 'org/apache/maven/**' // This needs to be unrelocated for Sponge
+    relocate("org.apache", "plan.org.apache") {
+        exclude "org/apache/logging/**"
+        exclude "org/apache/maven/**" // This needs to be unrelocated for Sponge
     }
-    relocate 'dagger', 'plan.dagger'
+
     // Don't relocate MySQL or SQLite since they are loaded with a isolated class loader
-    // relocate 'org.sqlite', 'plan.org.sqlite'
-    // relocate 'com.mysql', 'plan.com.mysql'
-    // relocate 'com.google.protobuf', 'plan.com.mysql.cj.x.google.protobuf'
-    relocate 'javax.inject', 'plan.javax.inject'
-    relocate 'com.github.benmanes', 'plan.com.github.benmanes'
-    relocate 'dev.vankka.dependencydownload', 'plan.dev.vankka.dependencydownload'
-
-    relocate 'com.maxmind', 'plan.com.maxmind'
-    relocate 'com.fasterxml', 'plan.com.fasterxml'
-    relocate 'com.zaxxer', 'plan.com.zaxxer'
-    relocate 'com.google.gson', 'plan.com.google.gson'
-    relocate 'com.google.errorprone', 'plan.com.google.errorprone'
-    relocate 'org.bstats', 'plan.org.bstats'
-
-    relocate 'org.eclipse.jetty', 'plan.org.eclipse.jetty'
-    relocate 'jakarta.servlet', 'plan.jakarta.servlet'
-    relocate 'javax.servlet', 'plan.javax.servlet'
-
-    relocate 'org.json.simple', 'plan.org.json.simple'
+    // relocate "org.sqlite", "plan.org.sqlite"
+    // relocate "com.mysql", "plan.com.mysql"
+    // relocate "com.google.protobuf", "plan.com.mysql.cj.x.google.protobuf"
+
+    relocate "dagger", "plan.dagger"
+    relocate "javax.inject", "plan.javax.inject"
+    relocate "jakarta.inject", "plan.jakarta.inject"
+    relocate "org.jspecify", "plan.org.jspecify"
```

Comment:
This is a new dependency from somewhere and seems to be annotations used in other libraries - does it need to be included?